dev/breeze/README.md

```diff
@@ -52,6 +52,6 @@ PLEASE DO NOT MODIFY THE HASH BELOW! IT IS AUTOMATICALLY UPDATED BY PRE-COMMIT.
 
 ---------------------------------------------------------------------------------------------------------
 
-Package config hash: c2bade421865f94fb91f652317d76f3686ad017067349265d8a1d465bbfc2f0796d570bf4777acfb31a7703f1ccb6b0a63c655b97cec9eb9fdeeb1676c804c02
```

Comment:
Why is this in here?

Reply:
Ah, not sure how it got added. I will revery this change

Reply:
@jedcunningham I am not sure why but looks like my breeze is forcing this change in and the pre commits fail. Maybe there is a reason for this?